static/js/components/dashboard/courses/test_util.js

```diff
@@ -0,0 +1,64 @@
+// @flow
+import moment from 'moment';
+
+import type { CourseRun } from '../../../flow/programTypes';
+import {
+  STATUS_CURRENTLY_ENROLLED,
+  STATUS_PASSED,
+  STATUS_NOT_PASSED,
+} from '../../../constants';
+
+export const makeRunCurrent = (run: CourseRun) => {
+  run.course_start_date = moment()
```

Comment:
The default format here won't be exactly the same as what's coming from the rest API, but for tests it probably won't matter